src/OpenTelemetry.Api/Trace/TracerProvider.cs

```diff
@@ -14,22 +14,40 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using System.Diagnostics;
 
 namespace OpenTelemetry.Trace
 {
     /// <summary>
-    /// TracerProvider is the entry point of the OTel API. It provides access to Tracers.
+    /// TracerProvider is the entry point of the OpenTelemetry API. It provides access to Tracers.
     /// </summary>
-    public abstract class TracerProvider
+    public class TracerProvider : IDisposable
     {
+        /// <summary>
+        /// Gets the default Tracer.
+        /// </summary>
+        public static TracerProvider Default = new TracerProvider();
```

Comment:
It's weird to have a public static field. Probably should be:
```csharp
 public static TracerProvider Default { get; } = new TracerProvider();
```
If you meant it to be settable, what's the use case for that?

Reply:
Its not meant to be settable (atleast in the current design). Picking your suggestion! thanks.